keygen/src/keygen.rs

```diff
@@ -388,6 +390,40 @@ fn app<'a>(num_threads: &'a str, crate_version: &'a str) -> Command<'a> {
                         .help("Overwrite the output file if it exists"),
                 )
         )
+        .subcommand(
+            Command::new("bls_pubkey")
+                .about("Display the BLS pubkey from a keypair file")
```

Comment:
nit: might want to specify that this is an ED25519 keyfile not a BLS keypair file and it's showing the derived pubkey

Reply:
Changed.